src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py

```diff
@@ -1379,6 +1380,65 @@ def test_import_iotype_without_omschrijving_generiek(self, *mocks):
 
         self.assertEqual(response.status_code, 302)
 
+    def test_import_zaaktype_with_bad_filenames(self, *mocks):
+        catalogus = CatalogusFactory.create(rsin="000000000", domein="TEST")
+        ZaakTypeFactory.create(
+            catalogus=catalogus,
+            vertrouwelijkheidaanduiding="openbaar",
+            zaaktype_omschrijving="bla",
+            datum_begin_geldigheid="2023-01-01",
+        )
+
+        url = reverse("admin:catalogi_catalogus_import_zaaktype", args=(catalogus.pk,))
+        response = self.app.get(url)
+
+        form = response.form
+        f = io.BytesIO()
+        f.name = "test.zip"
+        f.seek(0)
+
+        with zipfile.ZipFile(f, "w") as zip_file:
+            # zip_file.mkdir("some_dir")
```

Comment:
Added in python 3.11.

Not sure how to make directories in its current state. I don't think ZipFile was designed for this TBH.